packages/dashboard/tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+        "target": "es2020",
```

Comment:
should we lower the target to `ES5/ES2015` to make sure the app runs on older browsers?

Reply:
That's a good catch. I've checked and it looks like we can even just use Babel 7 to transpile our TypeScript to JavaScript, no need for having `ts-loader` (which would look at `tsconfig.json`) in the first place. This would respect our Babel config and the `tsconfig.json` will only act as a guideline for our IDE.